### DISCUSSION TITLE
How to use CLI Tool to upload more than 50 photos into Album

### DISCUSSION THREAD

**Discussion by srlamin19** [immich-app/immich#544] [General]
Unfortunately, for some reason, the new version (v1.26.0_36-dev) also brought a setback ,in the function of uploading photos to albums.

The instruction now is that, for those who want to create albums with more than 50 photos, use the CLI tool.

Could someone demonstrate how to use the CLI Tool? Is there somewhere with some kind of documentation? better than https://github.com/immich-app/immich/pull/379 or at least, how to dowgrade

![Screenshot from 2022-08-27 19-27-51](https://user-images.githubusercontent.com/111645772/187064106-83586381-37b6-4da8-8fe0-3d579090a9c0.png)

**Comment by alextran1502** [comment]
Here is the CLI repo

https://github.com/immich-app/CLI

There is an instruction in the readme file

**Comment by srlamin19** [comment]
Is it possible upload photos via CLI Tool to specific Album ? Or some parameter to create new album and upload? also based on folder name, generate new album (or upload the asset in the subfolder to same album name)

**Comment by srlamin19** [comment]
How can I send via CLI, the photos direct to specific the Album?

**Reply by alextran1502** [reply]
There is no feature like that for the CLI yet. Please put in an issue if you would like that to be implemented

**Reply by srlamin19** [reply]
Thanks Alex! https://github.com/immich-app/immich/issues/568
